README.md

```diff
@@ -82,6 +82,18 @@ A **Vagrant** configuration is included for development purposes. To use it, com
 - Run `vagrant ssh -c "cd /vagrant && foreman start"`
 - Open `http://mastodon.local` in your browser
 
+To set up **M1/2 MacOS** for native development, complete the following steps:
```

Comment:
Is any of this actually specific to M1/M2, or is this all just macOS in general? I think the only things different for Intel vs Apple Silicon are the ways you install the precursors to all this (homebrew, ruby version mgrs, etc)

I might remove entirely the M1/M2 qualification here, or I might rewrite as "Apple Silicon" so we don't need to update it next year for the M3 when (probably) no changes are actually needed.

Reply:
Great idea, will edit.